### PR TITLE
Added perwindowlayout contrib

### PR DESCRIPTION
**Pull request by wiedzmin**

### CODE REVIEW

File: util/perwindowlayout/perwindowlayout.lisp [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  "Disable layout switching"
+  (remove-hook *focus-window-hook* 'window-focus-changed)
+  (remove-hook *focus-group-hook* 'group-focus-changed))
+
+(defun toggle-window-layout ()
+  (let ((cur-window (current-window)))
+    (when cur-window
+      (let ((current-window-class (window-class cur-window)))
+        (if (string= current-window-class "Emacs")
+            (meta (kbd "C-\\"))
```
Comment (jorams) [inline util/perwindowlayout/perwindowlayout.lisp]: I think this could use a clarifying comment. Why are we sending `C-\` to Emacs?

Comment (wiedzmin) [inline util/perwindowlayout/perwindowlayout.lisp]: Definitely, I saw this code to often to get my eyes blurred . So maybe we may go further and use a defparameter, for example
`(defparameter *emacs-toggle-input-method-key* "C-\\")`
What do you think of it?

**Comment by dbjergaard** [comment]
Could you please include a little more info in the README? The other contrib modules are not good examples.  Ideally you would have a small paragraph describing the problem this solves, and then another small paragraph with an example (configuration in .stumpwmrc, and then how to use the code)

**Comment by wiedzmin** [comment]
Yes, indeed. BTW, maybe it wouldn't be bad if I improve some readmes for some other contribs that I use and find then unclear?

**Comment by dbjergaard** [comment]
Not only would I appreciate it, it would be a huge service to anyone looking to use these modules.  Please do!

**Comment by wiedzmin** [comment]
Is it anything wrong with this PR? If so, please provide some feeback.

**Comment by jorams** [comment]
Sorry, this slipped away.

While I personally don't have a use for this, it looks good to me.

**Pull request merged**